tests/system/CodeIgniterTest.php

```diff
@@ -56,7 +56,7 @@ public function testRunEmptyDefaultRoute()
 
 		$this->assertStringContainsString('Welcome to CodeIgniter', $output);
 	}
-
+	
```

Comment:
This file does not need any changes.
```suggestion
```

Reply:
Sorry was one of the places I tried todo a test for the filter from will correct